docs/topics/api/signing.rst

```diff
@@ -46,9 +46,11 @@ validation and you will be able to check its status.
             -g -XPUT --form 'upload=@build/my-addon.xpi'
             -H 'Authorization: JWT <jwt-token>'
 
-    :param addon-id the id for the add-on.
-    :param version: the version of the add-on.
-    :form upload: the add-on being uploaded.
+    :param addon-id: The id for the add-on.
```

Comment:
The docs are at http://pythonhosted.org/sphinxcontrib-httpdomain/#resource-fields